Q5Q support

luka177 opened the pull request.

TARKZiM committed:
sm7225-common: Build PhhIms

Change-Id: I7669e6d82cfc87e253fe4ad3174ab581b9b160a3

chaptsand committed:
sm8550-common: Build AIDL fingerprint HAL

Change-Id: Icafb53447a132517b916b83429753ab6d5810dab

luka177 committed:
sm8550-common: Add q5q support

Change-Id: Ic37d2ec06c5fff8b71ef2fb180ae7fa5e8972fb0

luka177 committed:
sm8550-common: Add bootconfig to cmdline
 While bootconfig allows to provide userspace config without taking space in cmdline,
 it only works with boot header v4, and recovery cant be newer then v2, so add bootconfig to cmdline,
 for recovery.

Change-Id: I2123b361b3c8e9097a94d80d762623aa8c3c236a

luka177 committed:
sm8550-common: Set BOARD_ROOT_EXTRA_FOLDERS

Change-Id: I6d3ef85bef4476b1ad57196d4c965d7dc52e309e

luka177 committed:
sm8550-common: proprietary-files: Remove blobs that are not common

Change-Id: I8668cd5715ae27abac6bcdbe841952e6c38024f7

luka177 committed:
sm8550-common: Import libshim_sensorndkbridge

Co-Authored-By: Tim Zimmermann <tim@linux4.de>
Change-Id: If5f03ada234d255cb52187e642423c08b972f0ff

luka177 committed:
sm8550-common: Copy init.recovery.qcom.rc to recovery root
As it is imported from root

Change-Id: I26c89841ca0012f5694896810816f8e1f43ad180

luka177 committed:
sm8550-common: Add missing keymaster and keymint libs

Change-Id: I727a779b30c6122f6c5d19d33d63922dfed9db0c

luka177 committed:
sm8550-common: Build libmeminfo

 Build libmeminfo from source

Change-Id: Id96370d9f2cb2e8974188cfa59b1c956dc849395

luka177 committed:
sm8550-common: Enable qti.media.c2audio if disabled

Change-Id: I1612c101978d3f1b4ef0afda6868a7723b355f13

luka177 committed:
sm8550-common: proprietary-files: Clean up

Change-Id: Ia91c45dbbc46cba9ed3ef798e90ce72d7541d7f8

chaptsand reviewed (commented):
Can be squashed with https://github.com/chaptsand/android_device_samsung_sm8550-common/pull/2/commits/788051f4006131d6f40d0abcbf29670b5b993a77

chaptsand reviewed (commented):
what's the use？

chaptsand reviewed (commented):
Is there any difference?

chaptsand reviewed (commented):
It's already in proprietary-files.txt, if you don't drop it. Build from source instead.

chaptsand reviewed (commented):
Have you tested for missing dependencies?

chaptsand commented on setup-makefiles.sh:
Can be squashed with https://github.com/chaptsand/android_device_samsung_sm8550-common/commit/788051f4006131d6f40d0abcbf29670b5b993a77

chaptsand commented on common.mk:
Is there any difference?

chaptsand commented:
> Is there any difference?

luka177 committed:
sm8550-common: Fic encryption

Change-Id: I37a64a081567ddbd94230c4b99d15f6430ac8d58

luka177 committed:
sm8550-common: Initial enforcing selinux

Change-Id: Ib401eb6090af24117a4009474dc76638d5d7d342

luka177 committed:
sm8550-common: Build SamsungDoze

Change-Id: I9a758f5617a71c9ab0f2680370e3e40710a6e0ae

luka177 committed:
sm8550-common: Use pre-built power hal

Change-Id: Id86ec09b10e6448d0fcc8f1cd002f9243ac49840

luka177 committed:
sm8550-common: Enable peak refresh rate toggle in Settings

Change-Id: Ia72385317c7255e3ba55624a7b8b5cd95da52587

luka177 committed:
[PERSONAL][DNM] sm8550-common: build OpenEUICC

Change-Id: I5d00e63a7121fadf30477868eab332dd06066989

luka177 committed:
sm8550-common: overlay: Do not set peak refrash rate

Change-Id: I852f48ca6e50eb16e53c30c969b6fbc14f83b659

luka177 committed:
sm8550-common: Switch to prebuilt composer

Change-Id: Iba6712db36dff553ee1b80864f65129cde6150c4

chirayudesai committed:
sm8550-common: Don't set BOARD_API_LEVEL

build/make/core/board_config.mk:1001: error: BOARD_API_LEVEL must not
set manully. The build system automatically sets this value..

Change-Id: If59ae9dea6cb13701d7217a1c0a355d4e07113eb

mikeNG committed:
sm8550-common: Build missing libraries for 14 QPR3

Change-Id: I6a84526ca1e9f4e5140c8068646433cab3fe85b2

luka177 committed:
sm8550-common: Add overlay to DEVICE_PACKAGE_OVERLAYS

Change-Id: Ic05ed4e7ceceb0e4c2758c48c3c736f5b51040e4

luka177 committed:
sm8550-common: Add sehradiomanager sepolicy

Change-Id: If28aa80a1b67d5977aacebd2858dea1339e3fdd7

luka177 committed:
sm8550-common: Ipdate proprietary blobs

Change-Id: I883cd83f6ac1f824ba2ef85bf59b8d0c59fdf395

luka177 committed:
sm8550-common: common: Move to samsung sensor service

This fixes proximity sensor

Change-Id: Idcfba31d8762cb443fe998aefe48cafe9226d92e

luka177 committed:
sm8550-common: releasetool: Flash vbmeta

Change-Id: I40491d208c7c012fe6296616d6a2061053e95a7a

luka177 committed:
sepolicy: Initial NFC/SecureElement policies

Change-Id: Ic8e886d9faf54690fb36207bb6bdccb5591fa28e

488315 committed:
sm8550-common: add gts9wifi support

luka177 committed:
Merge pull request #1 from 488315/patch-1

sm8550-common: add gts9wifi support

chaptsand commented:
Need a rebase
